python/ray/data/datasource/datasource.py

```diff
@@ -129,11 +130,11 @@ def get_name(self) -> str:
 
 @PublicAPI
 class Reader:
-    """A bound read operation for a datasource.
+    """A bound read operation for a :class:`~ray.data.Datasource`.
```

Comment:
This summary isn't meaningful to me. Don't the `ReadTask`s represent read operations?

Reply:
IIRC, the `Reader` is responsible for executing the read, which involves generating `ReadTask`s  for the corresponding `Datasource`